src/components/views/dialogs/ChatInviteDialog.js

```diff
@@ -351,17 +340,15 @@ module.exports = React.createClass({
         this.props.onFinished(true, addrTexts);
     },
 
-    _updateUserList: new rate_limited_func(function() {
```

Comment:
are you sure we want to remove the rate limiter?  doesn't it act as a safety-net to stop tight-looping recalculating the list as the user types?